charts/dim/values.yaml

```diff
@@ -85,6 +85,9 @@ processesworker:
       memory: 300M
   dim:
     applicationName: ""
+    # -- Sets the type of the status list which will be created for the issuer wallet
+    # -- valid types are:  StatusList2021, BitstringStatusList
+    statusListType: "BitstringStatusList"
```

Comment:
please change default value to StatusList2021, as the overall change to BitstringStatusList has been postponed.

Reply:
adjusted